test/dlc_tests/test_type_images.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please remove this from the files submitted in the PR.

Reply:
This is a draft PR. I will remove all the miscellaneous files when we are ready to convert the PR and before merging to master.